doc/ssl/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -16,8 +21,10 @@ Certificate Status Request functions
  long SSL_CTX_set_tlsext_status_arg(SSL_CTX *ctx, void *arg);
 
  long SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
+ long SSL_CTX_get_tlsext_status_type(SSL_CTX *ctx);
 
  long SSL_set_tlsext_status_type(SSL *s, int type);
+ long SSL_get_tlsext_status_type(SSL *s);
```

Comment:
Why long when the type is clearly int?

I have no idea why the setter returns long... but nevertheless I see no reason to repeat the mistake here, because the return types aren't even related (the setter, presumably, returns a success value, whereas the getter returns the type).